BaseTools/Source/Python/Workspace/WorkspaceCommon.py

```diff
@@ -123,6 +123,10 @@ def GetModuleLibInstances(Module, Platform, BuildDatabase, Arch, Target, Toolcha
     while len(LibraryConsumerList) > 0:
         M = LibraryConsumerList.pop()
         for LibraryClassName in M.LibraryClasses:
+        # MU_CHANGE [START]: Fix NULL LibraryClass Inclusion Issue
+            if LibraryClassName.startswith("NULL") and bool(M.LibraryClass):

```

Comment:
Although unlikely, should this check for an exact NULL match? For example, it shouldn't match a name like `NULLGpioLib`, right?

Reply:
The NULL instances are each given a unique library class name 'NULL<n>'. So if there is a single NULL instance defined for DXE_DRIVER, DXE_CORE, and UEFI_DRIVER then 3 NULL library names will be generated labelled NULL0, NULL1, and NULL2. We could split the NULL off and check that the remaining part is an integer, but that's probably not necessary here. The startswith("NULL") pattern is used to idetify NULL linked libraries in several places in Basetools.